llama-index-core/pyproject.toml

```diff
@@ -65,8 +65,8 @@ typing-extensions = ">=4.5.0"
 typing-inspect = ">=0.8.0"
 requests = ">=2.31.0"  # Pin to avoid CVE-2023-32681 in requests 2.3 to 2.30
 gradientai = {optional = true, version = ">=1.4.0"}
-asyncpg = {optional = true, version = "^0.28.0"}
-pgvector = {optional = true, version = "^0.1.0"}
+asyncpg = {optional = true, version = "^0.29.0"}
+pgvector = {optional = true, version = "^0.2.4"}
```

Comment:
Updated to match versions in `llama-index-integrations`; needed to work on both packages at once with Poetry.